robottelo/hosts.py

```diff
@@ -686,16 +686,20 @@ def virt_who_hypervisor_config(
             self.put_ssh_key(settings.server.ssh_key, hypervisor_ssh_key_name)
             # setup the ssh config and known_hosts files
             self.update_known_hosts(
-                self, hypervisor_ssh_key_name, hypervisor_hostname, user=hypervisor_user
+                hypervisor_ssh_key_name, hypervisor_hostname, user=hypervisor_user
             )
 
         # upload the virt-who config deployment script
         virt_who_deploy_directory = '/root/virt_who_deploy_output'
         virt_who_deploy_filename = f'{gen_alpha(length=5)}-virt-who-deploy-{config_id}'
         virt_who_deploy_file = f'{virt_who_deploy_directory}/{virt_who_deploy_filename}'
+        # create the virt-who directory on the broker VM
+        self.run(f'mkdir -p {virt_who_deploy_directory}')
+        # create the virt-who directory on satellite
+        satellite = Satellite(settings.server.hostname)
+        satellite.execute(f'mkdir -p {virt_who_deploy_directory}')
```

Comment:
Is there a difference between using `satellite.execute` and `ssh.command`?  Just out of curiosity.

Reply:
At this point, huge, but @JacobCallahan is trying to close that gap.

`robottelo.ssh` module uses paramiko.

`Host.execute` from broker uses ssh2-python, which is far faster. 

The pattern @swadeley has followed here is what we want moving forward - make SSH calls directly against the Host/Capsule/Satellite instance, instead of importing ssh methods and passing the host.

Reply:
`self.run` and `satellite.execute` are the same call, the methods are aliased right now.